src/app/shared/dialogs/dialogs-edit.component.html

```diff
@@ -1,5 +1,5 @@
 <mat-dialog-content>
-  <p i18n>Are you sure you want to delete the following {data.type, select,
+  <p i18n>Are you sure you want to {{data.changeType}} the following {data.type, select,
```

Comment:
Please familiarize yourself with translation for Angular and change this to use a select expression.

https://angular.io/guide/i18n